docs/get-started.md

```diff
@@ -77,7 +76,27 @@ In laravel, launch command as usual with your path:
 php artisan insights path/to/analyse
 ```
 
-## Specify composer file
+## Analyse multiple paths ![Version](https://img.shields.io/badge/Version-2.0.0-blue)
```

Comment:
In others docs page, we use `<Badge text="^1.x"/>` for the version. I'm not against use a badge from shields.io, but we should stay consistency. 
WDYT @olivernybroe, should we use a badge from shields.io everywhere ?

Reply:
I think we should stay with `Badge` at least for this pr. 

Using shield.io badges I think would be too heavy, as it would have to load a bunch of images, when a simple html element can be used instead.

Reply:
Ok, so I will change it to HTML badge.